### PR TITLE
Show correct age options while registration

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Description

While first login there was no max option shown for the second age range for users with existing account but missing birthday and consent. Also the grammar was not always clear for the range of ages during registration or first login.
## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-7589
## Changes

- fix code for correct age range
- change sentence structure in de and en for clearance
## Data Security <sub><sup>details [on Confluence](https://docs.schul-cloud.org/x/2S3GBg)</sup></sub>

No data security topics.
## Screenshots of UI changes

image

image

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -1462,7 +1462,7 @@
 		"label": {
 			"newPassword": "New password:",
 			"repeatYourNewPassword": "Repeat your new password:",
-			"under14YearsOld": "under 14 years old."
+			"under14YearsOld": "13 years or younger."
```
Comment (ABialoskorski) [inline locales/en.json:1465]: please change key to under13YearsOld

File: locales/de.json [hunk 2]
```diff
@@ -1462,7 +1462,7 @@
 		"label": {
 			"newPassword": "Neues Passwort: ",
 			"repeatYourNewPassword": "Wiederhole dein neues Passwort:",
-			"under14YearsOld": "unter 14 Jahre alt."
+			"under14YearsOld": "13 Jahre oder jünger."
```
Comment (ABialoskorski) [inline locales/de.json:1465]: please change key to under13YearsOld

**Comment by ABialoskorski** [comment]
we should also check age on backend site and in client site when age is calculated from date of birth

File: locales/en.json (hunk 1 above)
Comment (MartinSchuhmacher) [reply locales/en.json:1465]: key should stay like this as under 14 means 13 or younger

File: locales/de.json (hunk 2 above)
Comment (MartinSchuhmacher) [reply locales/de.json:1465]: key should stay like this as under 14 means 13 or younger

**Review by Parnswir — APPROVED** [review]

File: locales/en.json (hunk 1 above)
Comment (ABialoskorski) [reply locales/en.json:1465]: ah yes, my mistake ;)

**Pull request merged**